web_client/.eslintrc.js

```diff
@@ -5,12 +5,14 @@ module.exports = {
   },
   extends: [
     'plugin:vue/recommended',
+    // 'plugin:vuetify/recommended',
```

Comment:
Should this be removed?

Reply:
Sure, I can take that out.

Reply:
Removed in [3acbe32](https://github.com/OpenImaging/miqa/pull/645/commits/3acbe32edb71adb7fbb8d4fe1bc625d183d472fe)